vector/src/main/res/values/donottranslate.xml

```diff
@@ -36,4 +36,16 @@
         <item>@string/dark_theme</item>
         <item>@string/black_them</item>
     </string-array>
+
+    <string-array name="notification_area_show_codes">
+        <item>only_errors</item>
```

Comment:
order should be:
1. always (the default value)
2. message and errors
3. only_errors